staff/emma-cowdery/gamma-template/style.sass

```diff
@@ -0,0 +1,716 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:400,900')
+
+
+.body-home
+    margin: 0em
```

Comment:
when you want to set margin to 0 you don't need to add em, you could write margin: 0